plugins/trace.yaml

```diff
@@ -1,78 +1,47 @@
-
   platforms:
-    - uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.0-rc.1/kubectl-trace_0.1.0-rc.1_darwin_amd64.tar.gz
-      sha256: 52344e307daccb7aee559268c972bfd5b06de514066647ed6d4205aa0239df40
-      bin: kubectl-trace
-      files:
-        - from: ./kubectl-trace
-          to: kubectl-trace
-        - from: ./LICENSE
-          to: .
-      selector:
-        matchLabels:
-          os: darwin
-          arch: amd64
-    - uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.0-rc.1/kubectl-trace_0.1.0-rc.1_linux_amd64.tar.gz
-      sha256: cd151705bb5e8964aef5f4f8a6443dcdaef5ea66f5960371404e18b47edd485f
-      bin: kubectl-trace
-      files:
-        - from: ./kubectl-trace
-          to: kubectl-trace
-        - from: ./LICENSE
-          to: .
-      selector:
-        matchLabels:
-          os: linux
-          arch: amd64
-    - uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.0-rc.1/kubectl-trace_0.1.0-rc.1_windows_amd64.zip
-      sha256: 0d16f1e00ec990833986d3be4b2997ac48fd036fc3b30834b23eca30ee327853
-      bin: kubectl-trace.exe
-      files:
-        - from: ./kubectl-trace.exe
-          to: kubectl-trace.exe
-        - from: ./LICENSE
-          to: .
-      selector:
-        matchLabels:
-          os: windows
-          arch: amd64
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.1/kubectl-trace_v0.1.1_darwin_amd64.tar.gz
+    sha256: 1672bf6ea416763ce1f93bd49b74cf788664ba3be36808ef05505c1bc77f4e3b
+    bin: kubectl-trace
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.1/kubectl-trace_v0.1.1_linux_amd64.tar.gz
+    sha256: f132b535fedfec0d668a0e849b89b4ebb2e1773619125311a778273a90c74451
+    bin: kubectl-trace
+  - selector:
+      matchLabels:
+        os: linux
+        arch: 386
+    uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.1/kubectl-trace_v0.1.1_linux_386.tar.gz
+    sha256: a110d691383ff551264f534ddd2c69362a10b8715bb929661d77d9735df226e7
+    bin: kubectl-trace
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.1/kubectl-trace_v0.1.1_windows_amd64.zip
+    sha256: 5d92d3c926e28fa6d889d6b712eac44408a04df4d26b44022e3388b911a025fb
+    bin: kubectl-trace.exe
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/iovisor/kubectl-trace/releases/download/v0.1.1/kubectl-trace_v0.1.1_windows_386.zip
+    sha256: b49a49dc81f36d3d3aa3d2e4af777ed44b336c685bb94c213731d142bbcd5af7
+    bin: kubectl-trace.exe
+  shortDescription: Trace Kubernetes pods and nodes with system tools
+  description: |
+    Usage:
+      Read more documentation at: https://github.com/iovisor/kubectl-trace
```

Comment:
it would be better if you actually explain what the plugin does.
we already show this link (via `homepage:` field above) post-installation and "krew info" screen where this text also shows, so this text is redundant.

Reply:
I'll fix this in the PR that will replace this one.